README.md

```diff
@@ -100,6 +101,16 @@ Rules live in `review/src/ReviewConfig.elm` and can be modified if needed. These
 - add comments & questions
 - once approved, leave for the author to squash and merge
 
+### Bulk image resizing
+
+There is no hard limit on image size that can be uploaded. We want to keep
+this reasonable to make loading time as fast as possible for end users. So
+to bulk resize all images greater than '2000x2000' run:
+
+```
+mogrify -resize '2000x2000>' ./public/images/uploads/*.jpg
```

Comment:
I think take the upscale out of the example because we won't want that.

Reply:
Ah. Thanks. I misread your previous comment. I can't quite tell from the docs what the difference between without the > and having it are and ^ vs < but I think this is fine as it is now.

![image](https://github.com/geeksforsocialchange/teamwilder/assets/861172/fe5c68ee-9ffb-4535-83a5-00aef9ce90e1)